web/src/pages/index.jsx

```diff
@@ -1,23 +1,22 @@
-import React from 'react'
 import classNames from 'classnames'
 
 import Head from '@docusaurus/Head'
-import Nav from '../components/Nav/index'
-import Hero from '../components/Hero'
-import Features from '../components/Features'
-import Roadmap from '../components/Roadmap'
-import Testimonials from '../components/Testimonials'
 import ExampleWaspApps from '../components/ExampleWaspApps'
+import Faq from '../components/Faq.jsx'
+import Features from '../components/Features'
+import Footer from '../components/Footer'
+import Hero from '../components/Hero'
 import HowItWorks from '../components/HowItWorks'
-import ShowcaseGallery from '../components/ShowcaseGallery'
+import Nav from '../components/Nav/index'
 import Newsletter from '../components/Newsletter'
-import Faq from '../components/Faq'
-import Footer from '../components/Footer'
+import Roadmap from '../components/Roadmap'
+import ShowcaseGallery from '../components/ShowcaseGallery'
+import Testimonials from '../components/Testimonials'
 
 import waspCoverPhoto from '../../static/img/wasp_twitter_cover.png'
-import styles from './styles.module.css'
 import './index.css'
 import './preflight.css'
```

Comment:
Thought: I also have stuff set up so it reorders imports automatically, but I try not to commit those changes if they're not relevant, so I don't add noise to the review.

(No action needed, just sharing my approach)